src/main/java/study/step3/RacingGameProgress.java

```diff
@@ -0,0 +1,26 @@
+package study.step3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGameProgress {
+    private final static int INT_ONE = 1;
+
+    public static List<Round> gameStart(RacingGame racingGame, CarMoveCondition moveCondition) {
+        List<Round> rounds = new ArrayList<>();
+        Round round = initRound(racingGame.getCarEntries());
```

Comment:
```suggestion
        Round round = new Round(racingGame.getCarEntries());
```
가 아닌 메소드를 사용한 이유가 있나요 ?